eng/Packages.Data.props

```diff
@@ -134,8 +134,9 @@
     <PackageReference Update="Microsoft.Azure.SignalR.Management" Version="1.19.2" />
     <PackageReference Update="Microsoft.Azure.SignalR.Protocols" Version="1.19.2" />
     <PackageReference Update="Microsoft.Azure.SignalR.Serverless.Protocols" Version="1.9.0" />
-    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.36" />
-    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.36" />
+    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.37" />
+    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.37" />
+    <PackageReference Update="Microsoft.Azure.WebJobs.Host.Storage" Version="5.0.0" />
```

Comment:
Why is this reference needed?

Reply:
we need it to call [AddAzureStorageScaleServices](https://github.com/Azure/azure-webjobs-sdk/blob/55f3d9a7c8e0f38c9ad31b94585c9d395eeb7f57/src/Microsoft.Azure.WebJobs.Host.Storage/StorageServiceCollectionExtensions.cs#L39) in `ScaleHostEndToEndTest`.